docs/nodes/consensus-full-node.md

```diff
@@ -16,10 +16,10 @@ Consensus Layer.
 The following hardware minimum requirements are recommended for running the
 Consensus Full Node:
 
-* Memory: 8 GB RAM
-* CPU: Quad-Core
-* Disk: 250 GB SSD Storage
-* Bandwidth: 1 Gbps for Download/100 Mbps for Upload
+* Memory: **8 GB RAM**
+* CPU: **Quad-Core**
+* Disk: **500 GB SSD Storage**
```

Comment:
I disagree with @mindstyle85 and this 250GB is fine, especially since their storage load will be roughly 1/4 of a full node. I routinely run validators on less, and just prune or state sync. Perhaps we simply communicate that this is dependent upon pruning settings.

Reply:
ok we can leave it at that i guess, but it doesnt make sense for validator to be at 500gb and full storage node at 250gb, when its the latter that is usually set with pruning:nothing, which is what an rpc node should be about